lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -507,13 +507,12 @@ defmodule Kernel.SpecialForms do
   defmacro alias(module, opts), do: error!([module, opts])
 
   @doc """
-  Requires a given module to be compiled and loaded.
+  Tells Elixir you want to use the macros defined in a module.
```

Comment:
Hmmm.

I think the docs can use "requires" elsewhere, meaning "what `require` does". But using that verb in the description of `require` itself seems circular to me. I don't know what is "requiring" while defining "requiring", you see what I mean.